client/signup/steps/user/index.jsx

```diff
@@ -179,22 +179,19 @@ export class UserStep extends Component {
 	}
 
 	initGoogleRecaptcha() {
-		initGoogleRecaptcha(
-			'g-recaptcha',
-			'calypso/signup/pageLoad',
-			config( 'google_recaptcha_site_key' )
-		).then( ( result ) => {
-			if ( ! result ) {
-				return;
-			}
+		initGoogleRecaptcha( 'g-recaptcha', config( 'google_recaptcha_site_key' ) ).then(
+			( clientId ) => {
+				if ( clientId === null ) {
+					return;
+				}
 
-			this.setState( { recaptchaClientId: result.clientId } );
+				this.setState( { recaptchaClientId: clientId } );
 
-			this.props.saveSignupStep( {
-				stepName: this.props.stepName,
-				recaptchaToken: typeof result.token === 'string' ? result.token : undefined,
```

Comment:
Removing the `recaptchaToken` from this `saveSignupStep()` requires a little arcane knowledge about the old onboarding framework to understand. Each step can have some arbitrary data associated with it, and in this case we were saving the token as part of the step data. However it was never used. In the `submitForm()` method we overwrite it with the token from the `formSubmit` action.

https://github.com/Automattic/wp-calypso/blob/85ffdc493f40dde4261eb0ee5e0fa464a5e2c041/client/signup/steps/user/index.jsx#L278

This call to the `submit()` method ultimately ends up saving the `formSubmit` action as part of the steps data.